ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -115,36 +169,83 @@ private void showWelcomeMessage() {
   }
 
   private void executeStatements(final String queries) {
```

Comment:
Should we move this somewhere common and use it to execute run script from StatementExecutor as well?